controllers/configuration/backend/utils.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Do we need a test for utils.go.  Please avoid `utils` name https://go.dev/blog/package-names.